test/e2e/network/netpol/network_policy.go

```diff
@@ -623,6 +623,78 @@ var _ = SIGDescribeCopy("Netpol [LinuxOnly]", func() {
 			ValidateOrFail(k8s, model, &TestCase{FromPort: 81, ToPort: 81, Protocol: v1.ProtocolTCP, Reachability: reachabilityPort81})
 		})
 
+		ginkgo.It("should support denying of egress traffic on the client side (even if the server explicitly allows this traffic) [Feature:NetworkPolicy]", func() {
```

Comment:
Is this test case the counterpart of https://github.com/kubernetes/kubernetes/blob/0a839c6c3b9489716e216b4fc9dd2f9c17948c16/test/e2e/network/netpol/network_policy.go#L802 ?  If so, perhaps they should be placed together and be organized/commented similarly.  (Could also be a follow-on PR, not trying to drop the kitchen sink on this PR -- I think it's great!)

Reply:
Ah yeah, like move them under the same describe, ... 
```
ginkgo.Describe("policies can be controlled independently" func() {
 		ginkgo.It(" should support denying of egress traffic on the client side (even if the server explicitly allows this traffic) [Feature:NetworkPolicy]  based on PodSelector", func() {

                  }
               ginkgo.It("should enforce policies to check ingress and egress policies can be controlled independently based on PodSelector [Feature:NetworkPolicy]  based on PodSelector", func() {

                  }
} 
```

Because we want to confirm that these
-  works equivalently for namespaceSelectors
-  works equivalently for ports

@mattfenwick can you create a follow on issue summarizing ^ , ?

Reply:
just wrote it up here https://github.com/kubernetes/kubernetes/issues/97553

👍 👍